setup.py

```diff
@@ -10,7 +10,6 @@
 ]
 setup(
     name="dataflux-pytorch",
-    packages=["dataflux_core", "dataflux_pytorch"],
-    package_dir={"dataflux_core": "dataflux_client_python/dataflux_core"},
+    packages=find_packages(exclude=["*tests*"]),
```

Comment:
Yeah this looks promising until http://sponge2.corp.google.com/62d95cec-1727-41cc-9368-4a9e84bd2ee6 and you can see "E   ModuleNotFoundError: No module named 'dataflux_core'". Tricky business isn't it.